flink-tests/src/test/java/org/apache/flink/test/checkpointing/CheckpointAfterAllTasksFinishedITCase.java

```diff
@@ -75,6 +76,12 @@ public void setUp() {
         bigResult = sharedObjects.add(new CopyOnWriteArrayList<>());
     }
 
+    @AfterEach
+    public void teardown() {
+        IntegerStreamSource.latch = null;
```

Comment:
Instead of setting this variable to `null` in tearDown() and initializing it to `new CountDownLatch(1)` in the test, would it be simpler to just initialize it to `new CountDownLatch(1)` in setUp()?

Reply:
I think it would be better to initialize it if the tests actually use it. Besides, maybe someone would add a test that constructs a CountDownLatch initialized with a number bigger than 1 in this file.